client/src/components/Collections/common/CollectionEditView.vue

```diff
@@ -29,6 +29,23 @@
                     <suitable-converters-tab :suitable-converters="item" @clicked-convert="clickedConvert" />
                 </b-tab>
             </SuitableConvertersProvider>
+            <ConfigProvider v-slot="{ config }">
+                <b-tab v-if="config.enable_celery_tasks">
+                    <template v-slot:title>
+                        <font-awesome-icon icon="database" /> &nbsp; {{ l("Datatypes") }}
+                    </template>
+                    <datatypes-provider v-slot="{ item, loading }">
+                        <div v-if="loading"><loading-span :message="loadingString" /></div>
+                        <div v-else>
+                            <change-datatype-tab
+                                v-if="item && datatypeFromElements"
```

Comment:
Looks like an unnecessary `div` with a `v-else`? Should be a `v-else-if` on `change-datatype-tab`?